docs/src/main/tut/docs/config-cursors.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: docs
+title: Config Cursors
+---
+
+## {{page.title}}
+
+When a `ConfigReader` needs to be created from scratch, users need to implement a `from` method with the following
+signature:
+
+```scala
+def from(cur: ConfigCursor): Either[ConfigReaderFailures, A]
+```
+
+The `ConfigCursor` class is a wrapper for the raw `ConfigValue` provided by Typesafe Config. It provides an idiomatic,
+typesafe API for the most common operations needed while reading a config. In particular, it provides cast operations
+and key accesses that integrate neatly with the [PureConfig errors API](error-handling.html). When using cursors
+properly, most errors are automatically handled and filled with rich information about the location of the failure.
+
+We'll show how to implement our own `ConfigReader` for the following class:
+
+```tut:silent
+class Person(firstName: String, lastName: String) {
+  override def toString = s"Person($firstName $lastName)"
+}
+
+case class Conf(person: Person)
+```
+
+We intend our config to look like this:
+
+```tut:silent
+import com.typesafe.config.ConfigFactory
+
+val conf = ConfigFactory.parseString("person = { name: John Doe }")
```

Comment:
Nitpick: I think writing the config as `person.name: John Doe` would be less wonky. Feel free to ignore.